javascript-koans/koans/AboutApplyingWhatWeHaveLearnt.js

```diff
@@ -55,14 +55,31 @@ describe("About Applying What We Have Learnt", function() {
       }
     }
 
-    expect(sum).toBe(FILL_ME_IN);
+    expect(sum).toBe(233168);
   });
 
   it("should add all the natural numbers below 1000 that are multiples of 3 or 5 (functional)", function () {
+    var range = function (startNumber, endNumber) {
+      var resultArray = []
+      for(var i= 0; i<1000; i++){
+        resultArray.push(i)
+      }
+      // for loop starting at startNumber, ending at endNumber, pushing to resultArray each time
+      // resultArray.push(i)
+
+      return resultArray
+    }
+
+    var sum1 = range(0, 1000).reduce(function(sum2, number) {
```

Comment:
The usage of range validates my earlier comment. This will function right now because your loop above uses exactly 0 through 1000, but the functions definition seems like it intended to be used with any given range of values.